clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -1731,11 +1731,7 @@ static bool CheckConstexprDestructorSubobjects(Sema &SemaRef,
       SemaRef.Diag(Loc, diag::note_constexpr_dtor_subobject)
           << !FD << (FD ? FD->getDeclName() : DeclarationName()) << T;
     }
-
-    if (SemaRef.getLangOpts().CPlusPlus23)
-      return true;
-
-    return false;
+    return !!SemaRef.getLangOpts().CPlusPlus23;
```

Comment:
```suggestion
    return SemaRef.getLangOpts().CPlusPlus23;
```

Reply:
It doesn't let me do so because lambda has deduced type and `SemaRef.getLangOpts().CPlusPlus23` is an `int` but intended return type is `bool`. `!!` is a dirty hack to convert to bool then return the right value.

Reply:
i missed the lambda sorry. Looks fine (alternatively `-> bool` might be just as clean)